questionnaire/2020/questions.yaml

```diff
@@ -146,6 +146,43 @@
     - Proactive monitoring data analysis using AI/ML
     - We haven’t automated the detection of anomaly
 
+  - title: Telemetry – What system are you using to collect telemetry information from your network ? 
+    type: Multiple choice
+    id: telemetry-tool
+    responses:
+    - Telegraf - SNMP
+    - Telegraf - gNMI
+    - Telegraf - Exec/Execd
+    - SNMP Exporter
+    - BlackBox Exporter
+    - Node Exporter
+    - Logstash
+    - Beats
+    - Panoptes
+    - Kafka
+    - Internally developed collector
+    - Another open source project (not listed above)
+    - Network vendor specific collector
+    - We are using a traditional Network Monitoring solution
+
+  - title: Telemetry – What system are you using to store & visualize telemetry information from your network ? 
+    type: Multiple choice
+    id: telemetry-tool
```

Comment:
Does the id field need to be unique? Should this one be something like telemetry-store

Reply:
Good catch I made the change